stub-images/vddk-test-vmdk/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+load("@io_bazel_rules_docker//container:container.bzl", "container_push")
+
+
+
+container_image(
+    name = "vddk-test-image",
+    entrypoint = "mkdir -p /opt/testing && cp -f /libvddk-test-plugin.so /opt/testing/libvddk-test-plugin.so && cp -f /cirros.vmdk /opt/testing/nbdtest.img",
+    files = [
+        ":vddk-test-plugin",
+        ":cirros.vmdk",
```

Comment:
Do not add the Cirros image to the git repository! This needs to be downloaded from (stable) location during the build and verified against a checksum or GPG signature.

Reply:
sure , we need to add to add it to our main Workspace then, created PR here https://github.com/kubev2v/forklift/pull/40